extensions/sql-bindings/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -113,6 +114,13 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined):
 					.withAdditionalProperties(propertyBag).send();
 				return;
 			}
+
+			// Update sql extension package reference to latest SQL binding nuget package
```

Comment:
`Add latest sql extension package reference to project.` (Since the package reference might not be there yet)